index.md

```diff
@@ -5,8 +5,13 @@
 layout: home
 ---
 
-<div class="three-speakers col-md-12">
-  <h2>Featured Speakers</h2>
-  <p>Stay tuned we will announce our speakers soon.</p>
-  <p><a class="button-link" href="/schedule">See all the sessions</a></p>
+<div class="container">
+  <div class="row">
+    <div class="featured-speaker-wrapper clearfix">
```

Comment:
Would you like to instead add the `col-md-12` in this here? This is to also have the column directly after the `row` wrapper.